src/html/index.html

```diff
@@ -191,6 +191,8 @@ <h2 class="card__heading card__heading--iconless">Auto-login settings</h2>
               <li><a href="{{{ sso_dashboard_url }}}/logout" class="button button--full-width button--secondary">Log out of Mozilla</a></li>
             </ul>
           </div>
+          <!-- this input is intentionally hidden off screen to trigger LastPass -->
```

Comment:
Maybe we can make this a Mustache comment so that we only see it in the source? (with `{{! comment }}`)